.vsts-dotnet-ci.yml

```diff
@@ -66,25 +66,14 @@ jobs:
     name: Centos_x64_NetCoreApp31
     buildScript: ./build.sh
     container: CentosContainer
```

Comment:
Why are we removing the Centos matrixes?

Reply:
Those are now the default options so we don't need to specify them explicitly.